src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt

```diff
@@ -69,6 +71,26 @@ class LibraryService (
     return LibraryResult(null, LibraryStatus.LIBRARY_DOES_NOT_EXIST)
   }
 
+  fun addToLibraries(libraryIds: List<String>?, gameId: String): MultiLibraryResult {
+    val failedLibraries = mutableListOf<LibraryResult>()
+    if (libraryIds.isNullOrEmpty()) {
+      return MultiLibraryResult(emptyList(), MultiLibraryStatus.ALL_LIBRARIES_DO_NOT_EXIST)
```

Comment:
[nit] If the list is empty isn't that more of a bad request than a not found? I'd expect "all libraries do not exist" only if we're fed nonsense ids